integration_tests/sdk/param_test.py

```diff
@@ -88,6 +88,26 @@ def test_get_with_custom_parameter(client):
         param_doubled.get(parameters={"number": "NOT A NUMBER"})
 
 
+def test_implicitly_created_parameter(client):
+    @op
+    def func(foo):
+        return foo
+
+    result = func(2)
+    assert result.get() == 2
+    client._dag.must_get_operator(with_name="foo")
```

Comment:
In the interest of keeping our SDK tests at the user level, can't we simply verify this with `result.get(parameters={"foo": 10} == 10`, instead of inspecting the internal dag.